subcommittee/coding-guidelines/notetaker-role.md

```diff
@@ -0,0 +1,53 @@
+# Coding Guidelines Notetaker
+
+Thank you for taking notes for a session! The following are the steps you will follow.
+
+Reminder that we follow the consortium-wide agreed meeting policies. Please refer to them in the top-level [README](../../
+) under `Meeting Policies`.
+
+## Take meeting notes live in a Google Docs document
+
+During the meeting you'll take meeting minutes of participants' discussion in a Google Docs document that will be shared as the meeting begins.
+
+You'll add yourself as Notetaker for the meeting in the Google Doc.
+
+### Try to annotate live sections that are [info], [decision], and [todo]
+
+We use an annotation system to note those sections which are [info], [decision], or [todo] so that the meeting minutes can be easily searched.
+
+So in the meeting notes try to annotate live those points which come up that could fall into one of those categories.
+
+Feel free to speak up during the meeting to clarify if something has risen to the level of a [decision] or [todo] if unclear.
+
+## Submit the meeting minutes to the repo
+
+Following the meeting you will select all the contents of the Google Doc, right click, and then click `Copy as Markdown` on the menu that pops up.
```

Comment:
```suggestion
Following the meeting you will select all the contents of the Google Doc, right click, and then click `Copy as Markdown` on the menu that pops up. You have to make sure Markdown is enabled for the document. Enabling it can be done by checking **Tools -> Preferences -> Enable markdown**.
```